apps/mitt-konto/src/MittKonto/Main.purs

```diff
@@ -88,21 +92,40 @@ app = do
         }
   component "MittKonto" \_ -> React.do
     state /\ setState <- useState initialState
+    isPersonating /\ setPersonating <- useState' false
     let self = { state, setState }
-    pure $ render self logger
+        -- The user data in the search results isn't quite complete.
+        -- We do another fetch to get it all.
+        setActive result = case result of
+          Left _ -> setState $ setAlert $ Just
+                      { level: Alert.warning
+                      , title: "Laddningen misslyckades."
+                      , message: "Något gick fel."
+                      }
+          Right user -> do
+            Console.log $ "setstate active user " <> user.cusno
+            setState $ setActiveUser $ Just user
+            setPersonating true
```

Comment:
Just a minor observation: this will stay `true` even when I go back to my own account through search. Not a big deal and don't know if needs to be fixed. The only way to "reset" mitt konto is to refresh the page though.

Reply:
True, this was one thing I was hoping to have the new router for.